### PR TITLE
Logging v2

### PR DESCRIPTION
**Pull request by bbaldino**
jvb changes to adapt to the new logger in https://github.com/jitsi/jitsi-utils/pull/11

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/Conference.java [hunk 1]
```diff
@@ -231,21 +235,28 @@ public Conference(Videobridge videobridge,
...
+        Map<String, String> context = new HashMap<>();
+        context.put("confId", id);
+        context.put("gId", gid);
+        if (name != null)
         {
-            logger.setLevel(Level.WARNING);
+            context.put("name", name.toString());
         }
 
+        Level minLevel = enableLogging ? Level.ALL : Level.OFF;
```
Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/Conference.java]: I think this should be WARN or ERROR for health check conferences

File: src/main/java/org/jitsi/videobridge/Conference.java [hunk 2]
```diff
@@ -538,13 +549,14 @@ private void updateStatisticsOnExpire()
...
             StringBuilder sb = new StringBuilder("expire_conf,");
-            sb.append(logPrefix)
-                .append("duration=").append(durationSeconds)
+            sb.append("duration=").append(durationSeconds)
                 .append(",conf_completed=")
                     .append(videobridgeStatistics.totalConferencesCompleted)
                 .append(",has_failed=").append(hasFailed)
                 .append(",has_partially_failed=").append(hasPartiallyFailed);
-            logger.info(Logger.Category.STATISTICS, sb.toString());
+            //TODO(brian): bring back cateogry
```
Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/Conference.java]: I don't know if the category makes sense anymore. It was briefly used with a fluentd setup a long time ago.

File: src/main/java/org/jitsi/videobridge/Conference.java (hunk 1 above)
Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/Conference.java]: yup you're right, will fix.

File: src/main/java/org/jitsi/videobridge/Conference.java (hunk 2 above)
Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/Conference.java]: so do you think this should be removed completely? or just put on debug?

Comment (bgrozev) [reply src/main/java/org/jitsi/videobridge/Conference.java]: No, I think it's fine in INFO, I'm just saying we can remove the TODO and not worry about it (I don't think the conf_completed part is needed either).

**Pull request merged**